editor/layout/index.js

```diff
@@ -2,24 +2,35 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
+import './style.scss';
 import Header from 'header';
+import Sidebar from 'sidebar';
 import TextEditor from 'modes/text-editor';
 import VisualEditor from 'modes/visual-editor';
 
-function Layout( { mode } ) {
+function Layout( { mode, sidebarOpened } ) {
+	const className = classnames( 'editor-layout', {
+		'sidebar-opened': sidebarOpened
```

Comment:
As a modifier class, I might expect some boolean-ish prefix to the class name, maybe `has-open-sidebar` ?